src/Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorApp.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Column, GridColumns, Spacer, Text } from "@artsy/palette"
+import { ERROR_MESSAGES } from "Components/ErrorPage"
+import { ArtworkErrorApp_artworkError$data } from "__generated__/ArtworkErrorApp_artworkError.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { OtherWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppOtherWorks"
+import { RelatedWorksQueryRenderer } from "Apps/Artwork/Components/ArtworkErrorApp/ArtworkErrorAppRelatedWorks"
+
+interface ArtworkErrorAppProps {
+  artworkError: ArtworkErrorApp_artworkError$data
+}
+const ArtworkErrorApp = (props: ArtworkErrorAppProps) => {
+  const {
+    artworkError: { requestError, artwork },
+  } = props
+  const statusCode = requestError?.statusCode ?? 500
+  const headline = ERROR_MESSAGES[statusCode] ?? "Internal Error"
+
+  const artworkSlug = artwork?.slug
+
+  return (
+    <Box>
+      <Spacer y={6} />
+      <GridColumns>
+        <Column span={6} wrap>
+          <Text variant="xl">{headline}</Text>
+
+          <Text variant="xl" color="black60">
+            {statusCode}
+          </Text>
+
+          <Spacer y={2} />
+
+          {artworkSlug && (
+            <>
+              <OtherWorksQueryRenderer slug={artworkSlug} />
+              <Spacer y={2} />
+              <RelatedWorksQueryRenderer slug={artworkSlug} />
```

Comment:
Oh hello `RelatedWorks`  nice to [see you again](https://github.com/artsy/force/pull/11994) 👋🏽

Reply:
Yup! We can monitor the overall ES latency/health.